interp/interp_test.go

```diff
@@ -1038,3 +1041,34 @@ func TestRunnerOpts(t *testing.T) {
 		})
 	}
 }
+
+func TestContext(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*3)
```

Comment:
You're right, but remember that `go test` already has a timeout as a fallback for us. Which is configurable, too. So no need to do this ourselves.